pkg/store/consul/dsstore/auditing_store.go

```diff
@@ -0,0 +1,250 @@
+package dsstore
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/square/p2/pkg/audit"
+	"github.com/square/p2/pkg/ds/fields"
+	"github.com/square/p2/pkg/manifest"
+	"github.com/square/p2/pkg/types"
+	"github.com/square/p2/pkg/util"
+
+	klabels "k8s.io/kubernetes/pkg/labels"
+)
+
+type AuditLogStore interface {
```

Comment:
hmm it probably doesn't have to be, although i guess if we pretended we had godocs i don't think private types get documented?